_config/mimetypes.yml

```diff
@@ -756,7 +756,6 @@ SilverStripe\Control\HTTP:
     stl: application/vnd.ms-pki.stl
     str: application/vnd.pg.format
     stw: application/vnd.sun.xml.writer.template
-    sub: image/vnd.dvb.subtitle
```

Comment:
We may need to come up with a way of handling multiple mime types for a single extension